.github/workflows/auto-cherry-pick.yml

```diff
@@ -0,0 +1,263 @@
+                const cherryPickLabels = labels.filter(l => l.startsWith('cherry-pick/'));
+                
+                if (cherryPickLabels.length > 0) {
+                  shouldRun = true;
+                  commitSha = context.payload.pull_request.merge_commit_sha;
+                  targetBranches = cherryPickLabels.map(l => l.replace('cherry-pick/', ''));
+                  prNumber = context.payload.pull_request.number;
+                  prTitle = context.payload.pull_request.title;
+                }
+              }
+            } else if (context.eventName === 'issue_comment') {
+              // Comment on PR or issue
+              const comment = context.payload.comment.body;
+              const pickMatch = comment.match(/pick\s+((?:release-[\w.-]+(?:\s*,\s*)?)+)/i);
+              
+              if (pickMatch && context.payload.issue.pull_request) {
+                shouldRun = true;
+                
+                // Get PR details
+                const { data: pr } = await github.rest.pulls.get({
+                  owner: context.repo.owner,
+                  repo: context.repo.repo,
+                  pull_number: context.payload.issue.number
+                });
+                
+                if (pr.merged && pr.base.ref === 'develop') {
+                  commitSha = pr.merge_commit_sha;
+                  targetBranches = pickMatch[1].split(',').map(b => b.trim());
+                  prNumber = context.payload.issue.number;
+                  prTitle = pr.title;
+                } else {
+                  shouldRun = false;
+                  core.setFailed('PR must be merged and target develop branch');
+                }
+              }
+            }
+
+            core.setOutput('should-run', shouldRun);
+            core.setOutput('commit-sha', commitSha);
+            core.setOutput('target-branches', JSON.stringify(targetBranches));
+            core.setOutput('pr-number', prNumber);
+            core.setOutput('pr-title', prTitle);
+
+            console.log(`Should run: ${shouldRun}`);
+            console.log(`Commit SHA: ${commitSha}`);
+            console.log(`Target branches: ${JSON.stringify(targetBranches)}`);
+
+  cherry-pick:
+    needs: validate-trigger
+    if: needs.validate-trigger.outputs.should-run == 'true'
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        target-branch: ${{ fromJson(needs.validate-trigger.outputs.target-branches) }}
+      fail-fast: false
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Configure Git
+        run: |
+          git config --global user.name "github-actions[bot]"
+          git config --global user.email "github-actions[bot]@users.noreply.github.com"
+
+      - name: Set up GitHub CLI
+        run: |
+          echo "${{ secrets.GITHUB_TOKEN }}" | gh auth login --with-token
```

Comment:
Shouldn't need this. Whatever step that uses the `gh` cli, just pass the secret in the `env` for the step like:

```
      - name: List Open PRs
        run: |
          gh pr list <...>
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```